client/categories.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Contains the Category Class for NodeBB Client
+"""
+from __future__ import unicode_literals
+
+from openedx.features.openedx_nodebb_discussion.client import Client
+from openedx.features.openedx_nodebb_discussion.client.utils import save_category_relation_into_db
+
+
+class NodeBBCategory(Client):
+    """
+    Category Class responsible to make connection with NodeBB Client for categories related operations.
+    """
+    default_privileges = [
+        'groups:find',
+        'groups:read',
+        'groups:topics:read',
+        'groups:topics:create',
+        'groups:topics:reply',
+        'groups:topics:tag',
+        'groups:posts:edit',
+        'groups:posts:history',
+        'groups:posts:delete',
+        'groups:posts:upvote',
+        'groups:posts:downvote',
+        'groups:topics:delete',
+        'groups:posts:view_deleted',
+        'groups:purge',
+        'groups:moderate'
+    ]
+
+    default_groups = [
+        'registered-users',
+        'guests',
+        'spiders'
```

Comment:
why is this group in the default?

Reply:
It is Nodebb default group which came with the new installation. Whenever we create any category through API or through the admin panel of NodeBB these groups automatically assigned to the category.